handlers/interactions/buttonInteractions.ts

```diff
@@ -400,7 +403,10 @@ async function interactionButtonHandler(interaction: ButtonInteraction) {
 function removeIndex(arri: Array<string>, elem: string) {
```

Comment:
elem ? 
What does it even mean, could you make it more readable?

Reply:
users